vmdb/lib/tasks/test_metrics.rake

```diff
@@ -4,7 +4,7 @@
 #   original here: https://github.com/ManageIQ/metric_fu/blob/master/tasks/metric_fu.rake
 require_relative "./evm_test_helper"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
All .rake files are loaded by the rake loader, so you can't control the order.  I guess I don't understand what was wrong with it before?  Maybe a comment would make it better?